test/common/README.md

```diff
@@ -162,21 +162,21 @@ expectWarning({
 ```
 
 ### getArrayBufferViews(buf)
-* `buf` [&lt;Buffer>]
-* return [&lt;ArrayBufferView&#91;&#93;>]
+* `buf` [&lt;Buffer>][]
+* return [&lt;ArrayBufferView>][]\[\]
```

Comment:
Think the reference link might need to be updated instead for the 3 in this section to include the array modifier in the identifier

Reply:
The current way does work though

Reply:
My thinking in preferring this way is that the linked info is not about `ArrayBufferView[]`. It's about `ArrayBufferView`. So the link text reflects that.